SKYKitChat/Classes/Core/Cache/SKYChatCacheController.m

```diff
@@ -154,10 +139,8 @@ - (void)handleChangeEvent:(SKYChatRecordChangeEvent)event forMessage:(SKYMessage
 {
     switch (event) {
         case SKYChatRecordChangeEventCreate:
-            [self didSaveMessage:message error:nil];
-            break;
         case SKYChatRecordChangeEventUpdate:
-            [self didSaveMessage:message error:nil];
+            [self saveMessage:message completion:nil];
```

Comment:
This look a bit weird, since the delete event is still using `[self didDeleteMessage:message];`.